fairworkflows/fairstep.py

```diff
@@ -227,6 +239,42 @@ def is_script_task(self, value: bool):
         else:
             self.remove_attribute(RDF.type, object=namespaces.BPMN.ScriptTask)
 
+    @property
+    def code(self):
```

Comment:
I think you could use `self.get_attribute` for this instead of reimplemting it here (it is inherited from `RdfWrapper`). Or does it do something different?